dapp/src/components/claims/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./display";
+// export * as form from "./form";
```

Comment:
Should it be removed?

Reply:
> Should it be removed?

Yes, and done.

>     * There's no change to the package.json but the lock file changed -- should we agree on a npm version so we don't get those changes constantly? I'm personally using `7.7.6`

I'm not sure how to do that cleanly. I moved from node LTS (14.x.x) with npm version `6.3.0`, to the latest node (16.3.0) which got me npm version `7.15.1`. While that decreased the number of changes in the package-lock.json, it still had thousands of line changes. I think it runs deeper than just npm version, I believe that the package-lock changes every time an unpinned dependency of a dependency changes, which we don't really control.

Either way, I'm fine choosing a node (and corresponding npm) to standardize on, I'll use latest  to get a `7.x.x` npm, because node LTS is on `6.3.0`.
 
>     * You pushed a compiled WASM module from webpack, was that on purpose?

No, actually, I don't know how that happened. Should I remove it?

>     * `dapp/src/helpers/contract.ts` is empty

Good catch, removed.

> Otherwise, I tested it and there was no change from a user perspective so I guess that's a success.

The goal was to get it working the same way but be easier to extend, so hurrah!

Reply:
npm 7 seems good, least the lockfile version stays at 2, which was the biggest issue to me.